Whollet/Whollet/Source/Presentation/Wallet/WalletViewController.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+import Reusable
+
+enum WalletInitialState {
+    case load
+    case create
+}
+
+final class WalletViewController: UIViewController {
+    @IBOutlet private weak var descriptionText: UILabel!
+    @IBOutlet private weak var nextButton: UIButton!
+    var initialState: WalletInitialState?
+    private var viewModel: WalletViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configView()
+        bindViewModel()
+    }
+    
+    private func bindViewModel() {
+        viewModel = WalletViewModel()
+        viewModel.bsWalletState.bind { [weak self] value in
+            guard let self = self else { return }
+            if let state = value {
+                switch state {
+                case .inProgress:
+                    if self.initialState == .create {
+                        DispatchQueue.main.async {
+                            self.descriptionText.text = AppConstants.Strings.createICX
+                            self.nextButton.setTitle(AppConstants.Strings.loading, for: .normal)
+                            self.nextButton.backgroundColor = UIColor.MyTheme.primary.withAlphaComponent(0.5)
+                        }
+                    } else {
+                        DispatchQueue.main.async {
+                            self.descriptionText.text = AppConstants.Strings.loadWallet
+                            self.nextButton.setTitle(AppConstants.Strings.loading, for: .normal)
+                            self.nextButton.backgroundColor = UIColor.MyTheme.primary.withAlphaComponent(0.5)
+                        }
+                    }
```

Comment:
tương tự mấy cái khác
```suggestion
                         DispatchQueue.main.async {
                            self.descriptionText.text = self.initialState == .create ? AppConstants.Strings.createICX : AppConstants.Strings.loadWallet
                            self.nextButton.setTitle(AppConstants.Strings.loading, for: .normal)
                            self.nextButton.backgroundColor = UIColor.MyTheme.primary.withAlphaComponent(0.5)
                        }
```